recipe/meta.yaml

```diff
@@ -24,8 +23,8 @@ requirements:
   run:
     - python
     - numpy >1.20.0
-    - pytorch >=1.8.1
-    - lightning-utilities >=0.8.0
+    - pytorch >=1.10.0,<2.4.0
```

Comment:
same with lightning, you don't need the upper bound

Reply:
Done